keps/sig-architecture/2019-07-16-node-role-label-use.md

```diff
@@ -15,9 +15,10 @@ reviewers:
 approvers:
   - "@thockin"
   - "@derekwaynecarr"
+  - "@wojtek-t"
 creation-date: 2019-07-16
-last-updated: 2019-07-16
-status: implementable
+last-updated: 2020-06-01
+status: implemented
```

Comment:
@smarterclayton -- KEPs are marked `implemented` when they graduate to `Stable`. On the basis of https://github.com/kubernetes/enhancements/issues/1143#issuecomment-613159084, the KEP should have the `implementable` status.